app/uk/gov/hmrc/tariffclassificationfrontend/utils/Dates.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.utils
+
+import java.time.format.DateTimeFormatter
+import java.time.temporal.TemporalAccessor
+
+object Dates {
```

Comment:
I prefer to have all formatters in a single file

Reply:
Personally I prefer the separation. We can see what Pol thinks.